libs/helm/autocomplete/src/index.ts

```diff
@@ -1,11 +1,23 @@
 import { NgModule } from '@angular/core';
 import { HlmAutocomplete } from './lib/hlm-autocomplete';
+import { HlmAutocompleteCommand } from './lib/hlm-autocomplete-command';
+import { HlmAutocompleteSearch } from './lib/hlm-autocomplete-search';
+import { HlmAutocompleteSearchInput } from './lib/hlm-autocomplete-search-input';
 import { HlmAutocompleteTrigger } from './lib/hlm-autocomplete-trigger';
 
 export * from './lib/hlm-autocomplete';
+export * from './lib/hlm-autocomplete-command';
+export * from './lib/hlm-autocomplete-search';
+export * from './lib/hlm-autocomplete-search-input';
 export * from './lib/hlm-autocomplete-trigger';
 
-export const HlmAutocompleteImports = [HlmAutocomplete, HlmAutocompleteTrigger] as const;
+export const HlmAutocompleteImports = [
+	HlmAutocomplete,
+	HlmAutocompleteCommand,
+	HlmAutocompleteSearch,
+	HlmAutocompleteSearchInput,
```

Comment:
@ashley-hunter I copied over the command, command search and command search input for autocomplete. This way we don't need to override existing styles from command components and won't risk future styles to break autocomplete.

Reply:
Sounds good, makes sense!